molecule/cookiecutter/scenario/driver/azure/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/INSTALL.rst

```diff
@@ -6,11 +6,14 @@ Requirements
 ============
 
 * An Azure credentials rc file
-* azure sdk
 
 Install
 =======
 
+Please refer to the `Virtual environment`_ documentation for installation best practices.
+
+.. _Virtual environment: https://virtualenv.pypa.io/en/latest/
```

Comment:
I see you refer to this page multiple time. I'd prefer having an intersphinx reference instead.

Reply:
`Intersphinx` doesn't work here since it is out of the `docs/` folder AFAIK.

Reply:
Ok